src/WebServerManager.cpp

```
@@ -614,6 +615,26 @@ void WebServerManager::handleApiSpoolmanLink() {
     _server.send(200, "application/json", "{\"success\":true}");
 }
 
+void WebServerManager::handleApiSpoolmanPendingLink() {
+    _server.sendHeader("Access-Control-Allow-Origin", "*");
+
+    StaticJsonDocument<128> doc;
```

Clang (14.0.6)

[warning] variable 'doc' is not initialized

(cppcoreguidelines-init-variables)